sdk/program/src/vote/state/mod.rs

```diff
@@ -302,7 +338,7 @@ impl VoteState {
     /// Upper limit on the size of the Vote State
     /// when votes.len() is MAX_LOCKOUT_HISTORY.
     pub const fn size_of() -> usize {
-        3731 // see test_vote_state_size_of.
+        3762 // see test_vote_state_size_of.
```

Comment:
This broke the JavaScript client's ability to create new accounts. See https://github.com/solana-labs/solana-web3.js/pull/1306